codespell_lib/data/dictionary.txt

```diff
@@ -2383,7 +2391,9 @@ challange->challenge
 challanged->challenged
 challanges->challenges
 challege->challenge
+champain->champagne
```

Comment:
also "champaign"

Reply:
the city?

Reply:
You are right, there is only the city. The drink is Champagne. I mixed it up.